### PR TITLE
[DO NOT MERGE] update webapi to 4.6 and fix send trace

### PR DESCRIPTION
**Pull request by johnataylor**

### CODE REVIEW

File: samples/csharp_webapi/13.core-bot/Microsoft.LuisRecognizer.schema [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (sgellock) [inline samples/csharp_webapi/13.core-bot/Microsoft.LuisRecognizer.schema]: Why do we have .schema files for this sample?

File: samples/csharp_webapi/13.core-bot/Schemas/Microsoft.IDialog.schema [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (sgellock) [inline samples/csharp_webapi/13.core-bot/Schemas/Microsoft.IDialog.schema]: and this one too?

File: samples/csharp_webapi/13.core-bot/Microsoft.LuisRecognizer.schema (hunk 1 above)
Comment (johnataylor) [reply samples/csharp_webapi/13.core-bot/Microsoft.LuisRecognizer.schema]: that was an accident!!!
i just pulled the sample, restored packages, fixed a build issue in the sending trace and now i see this schema stuff?

Comment (johnataylor) [reply samples/csharp_webapi/13.core-bot/Microsoft.LuisRecognizer.schema]: do you know where it comes from?

Comment (johnataylor) [reply samples/csharp_webapi/13.core-bot/Microsoft.LuisRecognizer.schema]: so my assumption was this was correct - i.e. something in the packages that nuget restore added

File: samples/csharp_webapi/13.core-bot/Schemas/Microsoft.IDialog.schema (hunk 2 above)
Comment (johnataylor) [reply samples/csharp_webapi/13.core-bot/Schemas/Microsoft.IDialog.schema]: OK so the schema (and the  project file updates) come from the NuGet package.

**Comment by johnataylor** [comment]
OK so we are evaluating whether we need to check these Schema files in. They come from "content" in NuGet. It seems harmless but noisy.

**Comment by johnataylor** [comment]
we have the update to teh send trace in already - this is now just the package update

**Comment by johnataylor** [comment]
closing this one in favor of a new one that goes straight to 4.6.1

**Pull request closed without merging**